featurex/tests/test_extractors.py

```diff
@@ -95,6 +96,14 @@ def test_saliency_extractor():
         sf = tl.data['SaliencyExtractor'].data['frac_high_saliency']
         assert np.isclose(sf,0.27461971)
 
+def test_tesseract_extractor():
+    pytest.importorskip('pytesseract')
```

Comment:
Addressed: tesseract can easy be installed using `sudo apt-get install tesseract-ocr`